### DISCUSSION TITLE
CDS energy in CPCM-X

### DISCUSSION THREAD

**Discussion by choutkaj** [grimme-lab/xtb#965] [General]
Hi all.

I'm running CPCM-X calculations with a bleeding edge version of xtb (fc056d1). I am not quite sure which energies are the correct CPCM-X energies. The ones from the SUMMARY box (Gelec and Gsasa), or the ones from the "CPCM-X post-SCF solvation evaluation" section (dG_solv)?

Furthermore, is there a printout of just the nonpolar solvation energy (CDS term)? Is this printed out as the Gsasa?

Thanks.

Here is a sample output: [protein.log](https://github.com/grimme-lab/xtb/files/14298710/protein.log)

**Comment by lukaswittmann** [comment]
The correct one should be the `total free energy (dG)` and the associated solvation energy is `solvation free energy (dG_solv):`.

**Reply by choutkaj** [reply]
OK. Thanks.
